src/htaccess

```diff
@@ -723,6 +723,7 @@ AddEncoding br .br
     # Serve a different manifest to iOS
     RewriteCond %{REQUEST_URI} =/manifest-webapp-6-2018.json
     RewriteCond %{HTTP_USER_AGENT} "iPhone|iPad"
```

Comment:
Should we remove this line?

Reply:
No, we can't. The condition following it is negated so if we would remove that line it would match all (even Android, etc.) devices except for newer iOS devices. The line following this is only concerned about the version string, not the device type, we could remove the (iPhone|iPad) from the following line but would still need to keep this.

Reply:
Got it, thanks. I’ll merge this on Monday and we can get some Beta testing in.